tests/check_ghostty_zig_workflows.py

```
@@ -1,60 +1,436 @@
 #!/usr/bin/env python3
-"""Require Ghostty submodule initialization before every Zig manifest consumer."""
+"""Require Ghostty initialization before workflow steps execute Zig consumers."""
 
 from __future__ import annotations
 
+import argparse
 import re
 import sys
+from dataclasses import dataclass
 from pathlib import Path
+from typing import Iterator
```

Ruff (0.16.0)

[warning] Import from `collections.abc` instead: `Iterator`

Import from `collections.abc`

(UP035)